src/elm/Components/Form.elm

```diff
@@ -183,13 +193,17 @@ viewButton { id_, msg, text_, classList_, disabled_ } =
         [ text text_ ]
 
 
+{-| viewSubtitle : renders a subtitle with with temporary content.
```

Comment:
`with with` typo and not sure i understand what temporary content means

Reply:
corrected typo, but plan to go back to clarify "temporary content" comments.